networks/suzuka/suzuka-full-node/src/partial.rs

```diff
@@ -306,7 +307,17 @@ where
 			}
 			BlockCommitmentEvent::Rejected { height, reason } => {
 				debug!("Commitment rejected: {:?} {:?}", height, reason);
-				// TODO: block reversion
+				let current_head_height = executor.get_block_head_height().await?;
+				if height > current_head_height {
+					// Nothing to revert
+				} else {
+					match executor.revert_block_head(height - 1) {
```

Comment:
If this is the correct usage, might it be better to name this `revert_block_head_to`?